modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/AbstractSnapshotVerificationTask.java

```diff
@@ -100,19 +98,60 @@ public abstract class AbstractSnapshotVerificationTask extends
 
     /**
      * @param name Snapshot name.
-     * @param path Snapshot directory path.
-     * @param incIdx Incremental snapshot index.
      * @param constId Snapshot metadata file name.
-     * @param groups Cache groups to be restored from the snapshot. May be empty if all cache groups are being restored.
-     * @param check If {@code true} check snapshot before restore.
+     * @param args Check snapshot parameters.
+     *
      * @return Compute job.
      */
-    protected abstract ComputeJob createJob(
-        String name,
-        @Nullable String path,
-        int incIdx,
-        String constId,
-        Collection<String> groups,
-        boolean check
-    );
+    protected abstract AbstractSnapshotPartitionsVerifyJob createJob(String name, String constId, SnapshotPartitionsVerifyTaskArg args);
```

Comment:
1. Misprint - constId -> consId
2. Javadocs says that it is "metadata filename", whether the docs or the variable should be renamed